BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -36,7 +36,7 @@ public final class BlueprintView: UIView {
 
     private let rootController: NativeViewController
 
-    private var cachedIntrinsicContentSize: CGSize? = nil
+    private var sizesThatFit: [SizeConstraint: CGSize] = [:]
```

Comment:
We _could_; but those are currently all short-lived because they depend on the exact element state at call time (eg, the subtrees are stored by element index); so it seemed to make more sense to introduce a top-level cache that holds "inert" measurement values. Eg, we'd still need to invalidate the `CacheTree` in all the same places we do here, and keeping one around seems to potentially invite mis-use when all we really care about here is the top-level size; not digging into child sizes.

Reply:
> we'd still need to invalidate the CacheTree in all the same places we do here

Yep, that's exactly what I had in mind. The `CacheTree` has an advantage in that a top-level miss can still have nested cache hits.

This is fine as-is though; I'll explore converting to `CacheTree` later.